marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I agree with @graimundo and @pamval that including a kettle-core is a very bad way to go, worse than duplicating code. This would seem like a good candidate to be a module in a "pentaho-util" repo, assuming this is actually used across several projects. @lgrill-pentaho you agree?